frontend/package.json

```diff
@@ -38,7 +38,10 @@
     "debug-test-suite": "TS_NODE_COMPILER_OPTIONS='{\"module\":\"commonjs\"}' node -r ts-node/register --inspect-brk ./node_modules/.bin/protractor integration-tests/protractor.conf.ts",
     "analyze": "NODE_ENV=production NODE_OPTIONS=--max-old-space-size=4096 ts-node -O '{\"module\":\"commonjs\"}' ./node_modules/.bin/webpack --mode=production --profile --json | sed '0,/^{/s/^[^{].*//g' > public/dist/stats.json && ts-node -O '{\"module\":\"commonjs\"}' ./node_modules/.bin/webpack-bundle-analyzer --mode static -r public/dist/report.html public/dist/stats.json",
     "prettier-all": "prettier --write '**/*.{js,jsx,ts,tsx,json}'",
-    "gql-generate": "graphql-codegen"
+    "gql-generate": "graphql-codegen",
+    "i18n-to-pot": "node ./i18n-scripts/i18n-to-pot.js",
+    "po-to-i18n": "node ./i18n-scripts/po-to-i18n.js",
+    "i18n": "./i18n-scripts/build-i18n.sh && node ./i18n-scripts/consolidate-public-folders && node ./i18n-scripts/set-english-defaults.js"
```

Comment:
Again, you seem to be overloading this PR.  PR should just be for cypress-cucumber framework and i18n should be a follow on PR. CC @spadgett

Reply:
Clean up the codeis done